commcare_connect/events/views.py

```diff
@@ -0,0 +1,34 @@
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from rest_framework import serializers, status
+from rest_framework.generics import ListCreateAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from .models import Event
+
+
+class EventSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Event
+        fields = ["date_created", "event_type", "user", "opportunity"]
```

Comment:
I think this will need some kind of event id from the phone, so that we can communicate success info for individual items in the list as described below

Reply:
Agreed, I was thinking to use the timestamp as such ID, but a UUID would be nice.

Reply:
Responded here https://github.com/dimagi/commcare-connect/pull/349/files#r1701902592